scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py

```diff
@@ -240,11 +280,47 @@ def fix_file(filename: str, qgis3_compat: bool) -> int:
             elif (isinstance(node, ast.ImportFrom) and node.module and node.module.startswith("PyQt5.")):
                 fix_pyqt_import.append(Offset(node.lineno, node.col_offset))
 
-    if not fix_qvariant_type and not fix_pyqt_import and not fix_qt_enums and not rename_qt_enums and not member_renames and not function_def_renames:
+    if not fix_qvariant_type and not fix_pyqt_import and not fix_qt_enums and not rename_qt_enums and not member_renames and not function_def_renames and not custom_updates and not extra_imports:
         return 0
 
     tokens = src_to_tokens(contents)
     for i, token in reversed_enumerate(tokens):
+        if token.offset in import_offsets:
+            end_import_offset = Offset(*import_offsets[token.offset][-2:])
+            assert tokens[i].src == 'from'
+            token_index = i + 1
+            while not tokens[token_index].src.strip():
+                token_index += 1
+
+            module = ''
+            while tokens[token_index].src.strip():
+                module += tokens[token_index].src
+                token_index += 1
+
+            if extra_imports.get(module):
```

Comment:
Can we just display a message when extra_imports doesn't contain the module. Even if that cannot happen in the QDateTime scenario, it could be the case if we broader use extra_imports (I'm thinking for instance of import that move from one module to another)

Reply:
Good idea, done